pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -367,7 +389,7 @@ def get_treeinfo_data(remote, remote_url):
             path = f"{repodata}/"
```

Comment:
This feels wrong - get_repomd_file() already appens repodata/repomd.xml to the incoming URL.